src/classes/HH_NameSpec_TEST.cls

```diff
@@ -95,18 +95,17 @@ private with sharing class HH_NameSpec_TEST {
     static testMethod void TwoContactSameLastname() {
         if (strTestOnly != '*' && strTestOnly != 'TwoContactSameLastname') return;
                 
-        list<Contact> listCon = new list<Contact>();
-        Contact c1 = new Contact (
-            Salutation = 'Skipper',
-            FirstName = 'John',
-            LastName = 'Doe'
-        );
-        listCon.add(c1);
-        Contact c2 = new Contact (
-            FirstName = 'Jane',
-            LastName = 'Doe'
-        );
-        listCon.add(c2);
+        Contact[] listCon = new Contact[] {
```

Comment:
another example of an unnecessary change in a test that wasn't substantively changed.